src/Agent.Listener/Configuration/ConfigurationProvider.cs

```diff
@@ -263,18 +263,18 @@ public void UpdateAgentSetting(AgentSettings settings)
         {
             ArgUtil.NotNull(_machineGroupServer, nameof(_machineGroupServer));
 
-            DeploymentMachineGroup machineGroup = (await _machineGroupServer.GetDeploymentMachineGroupsAsync(projectName, machineGroupName)).FirstOrDefault();
+            var deploymentGroup = (await _machineGroupServer.GetDeploymentGroupsAsync(projectName, machineGroupName)).FirstOrDefault();
```

Comment:
I am assuming the scope of this PR is to consume the new API only. 
Renaming the user facing inputs and logs will be taken in another PR.